printer_mods/Mellow/V0_Fly-Gemini/readme.md

```diff
@@ -0,0 +1,4 @@
+# V0 adaptation model of Fly-Gemini
+
+## Fixed bracket
```

Comment:
Please add a Parts BOM including what screws are needed to use the mounts and how many.

Other instalation instructions would also be useful. does this attach with VHB? Does the user need to drill holes for screws?